src/board-explorer-app/board-explorer-app.html

```diff
@@ -1185,9 +1183,13 @@
          href += "#" + board;
       }
 
+      if (this.category) {
+        href += "&" + this.category.slug;
```

Comment:
Hmmm. Let's work through the URI parsing; I'm hesitant to have to introduce a token that isn't has meaning which isn't intuitively guessed by someone looking at the URL. 

If the set of filters is always additive of anything in the list of filters, then if we had a category defined as:

```json
{ "power-category" = [ "VBAT", "POWER" ] }
```
The following would all be equivalent:

```
/power-category
/power-category,VBAT
/POWER,power-category
/VBAT,POWER
/POWER,power-category,VBAT
```

If the user **CTRL+clicks** an item to toggle it's inclusion in the filter list, it would be added or removed from the URI list. Removing VBAT from an URL that includes `power-category` would not end up hiding the filter in the UI (`VBAT` is still enabled in the `power-category` filter)

I had been considering defining the URI path format as:

```bnf
<path> ::= <category-path> | <filter-path> | <category-filter-path> | ""
<category-path> ::= "/" <category>
<filter-path> ::= "/" <filter>
<category-filter-path> ::= <category-path> <filter-path>
<filter> ::= <refdes> | <refdes> "," <filter> | ""
<category> ::= <string-match-from-getCategories>
```

Doing the above makes the URL possibly look more appropriate, but it doesn't change the additive property of the items listed in the URL.